lib/avromatic/model/types/decimal_type.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'bigdecimal'
+require 'bigdecimal/util'
```

Comment:
Why is this required?

Reply:
This provides `to_d` method for other types like `Float`, `Integer`, `String`, etc.